### PR TITLE
BorderedButton added

### PR DESCRIPTION
**Pull request by lutfullahkamilcaglar**

### CODE REVIEW

File: lib/components/buttons/bordered_button.dart [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+  const BorderedButton({
+    Key? key,
+    required this.onPressed,
+    this.child,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return ElevatedButton(
```
Comment (Logaritma26) [inline lib/components/buttons/bordered_button.dart]: can we use OutlinedButton, instead of Elevated

Comment (lutfullahkamilcaglar) [reply lib/components/buttons/bordered_button.dart]: ElevatedButton changed to OutlinedButton

File: lib/resource/theme_color.dart [hunk 2]
```diff
@@ -17,5 +17,8 @@ abstract class ThemeColor {
   
   static const Color Secondary = Color.fromRGBO(241, 245, 249, 1);
   static const Color Destructive = Color.fromRGBO(244, 3, 5, 1.0);
-  
+
```
Comment (Logaritma26) [inline lib/resource/theme_color.dart:20]: why new lines ? remove leftovers

File: lib/components/buttons/outline_button.dart [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    Key? key,
+    required this.onPressed,
+    this.child,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return OutlinedButton(
+      style: OutlinedButton.styleFrom(
+        backgroundColor: ThemeColor.White,
```
Comment (Logaritma26) [inline lib/components/buttons/outline_button.dart]: is background color still required while using `OutlinedButton` ?

File: lib/components/buttons/outline_button.dart [hunk 4]
```diff
@@ -0,0 +1,25 @@
+import 'package:f_kit_x/resource/theme_color.dart';
+import 'package:flutter/material.dart';
+
+class OutlineButton extends StatelessWidget {
+  final void Function()? onPressed;
+  final Widget? child;
```
Comment (Logaritma26) [inline lib/components/buttons/outline_button.dart]: child should be non-nullable

File: lib/components/buttons/outline_button.dart [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return OutlinedButton(
+      style: OutlinedButton.styleFrom(
+        foregroundColor: ThemeColor.Primary,
+      ),
+      onPressed: onPressed,
+      child: child,
```
Comment (Logaritma26) [inline lib/components/buttons/outline_button.dart]: add padding to child

**Review by Logaritma26 — APPROVED** [review]

**Pull request merged**